pages/calendar/components/Calendar/WeeklyCalendar/index.tsx

```diff
@@ -35,10 +71,20 @@ export default function WeeklyCalendar() {
           ))}
         </div>
         <div className={cn("time-container")}>
-          {dates.map((date, idx) => (
-            <div key={idx} className={cn("time-column")}>
-              {HOURS.map((hour) => (
-                <div key={hour} className={cn("time-block")}></div>
+          {dates.map((date, dayIndex) => (
+            <div key={dayIndex} className={cn("time-column")}>
+              {HOURS.map((hour, hourIndex) => (
+                <div
+                  key={hour}
+                  className={cn("time-block", {
+                    dragged:
+                      draggedHoursIndex.includes(hourIndex) && dragStartDayIndex === dayIndex,
+                  })}
+                  data-index={hourIndex}
+                  onPointerDown={() => handleDragStart(dayIndex, hourIndex)}
+                  onPointerMove={() => handleDragMove(hourIndex)}
+                  onPointerUp={handleDragEnd}
+                ></div>
```

Comment:
```suggestion
                />
```
p4: 안에 내용이 없다면 `self closing`해도 괜찮을 것 같아요!